test/python/fe_api/causal_conv1d_bulk/test_causal_conv1d_bulk_sm100.py

```
@@ -0,0 +1,563 @@
+os._exit(9)
+"""
+
+
+@pytest.mark.parametrize("case", ["start_nonzero", "end_mismatch", "non_increasing"])
+def test_invalid_cu_seqlens_fail_closed_in_fresh_process(case):
+    # A PTX trap poisons its CUDA context, so isolate each metadata class and
+    # first prove that the same compiled object accepts valid boundaries.
+    source_cudnn = Path(__file__).resolve().parents[4] / "python" / "cudnn"
+    environment = os.environ.copy()
+    environment["PYTHONNOUSERSITE"] = "1"
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)